armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -352,7 +352,7 @@ def fromReactor(self, reactor: reactors.Reactor):
         self.symmetry = reactor.core.symmetry
 
         cycleNodeStamp = f"{reactor.p.cycle:03d}{reactor.p.timeNode:03d}"
-        if self.savePhysicsFilesList is not None:
+        if self.savePhysicsFilesList:
```

Comment:
lol, what a funny bug. Someone was trying to _avoid_ short-circuiting boolean evaluation but they ended up allowing `False` values to be seen as `True`.

Nice catch!

Reply:
Yeah this was causing a bug down stream in our plugins as I was running some tests.